dev/com.ibm.ws.microprofile.faulttolerance.2.0/src/com/ibm/ws/microprofile/faulttolerance20/state/impl/CircuitBreakerStateImpl.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.faulttolerance20.state.impl;
+
+import com.ibm.ws.microprofile.faulttolerance.impl.CircuitBreakerImpl;
+import com.ibm.ws.microprofile.faulttolerance.spi.CircuitBreakerPolicy;
+import com.ibm.ws.microprofile.faulttolerance20.impl.MethodResult;
+import com.ibm.ws.microprofile.faulttolerance20.state.CircuitBreakerState;
+
+public class CircuitBreakerStateImpl implements CircuitBreakerState {
+
+    private final CircuitBreakerImpl circuitBreaker;
+
+    public CircuitBreakerStateImpl(CircuitBreakerPolicy policy) {
+        circuitBreaker = new CircuitBreakerImpl(policy);
```

Comment:
consider writing own circuit breaker code to remove FS dependency

Reply:
Raised #5538